app/assets/stylesheets/mixins.scss

```diff
@@ -20,6 +20,14 @@
   }
 }
 
+%consul-gradient {
+  background: #36424a;
+  background: -moz-linear-gradient(-45deg, #36424a 0%, #3a778b 100%);
+  background: -webkit-linear-gradient(-45deg, #36424a 0%,#3a778b 100%);
+  background: linear-gradient(135deg, #36424a 0%,#3a778b 100%);
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#36424a', endColorstr='#3a778b',GradientType=1 );
```

Comment:
I think we can remove this line :thinking:. Internet Explorer 10 (released in 2012) already supports the linear-gradient function.